src/components/renderers/layouts/categories/categorization.directive.ts

```diff
@@ -15,30 +15,51 @@ class CategorizationController  extends AbstractLayout {
     constructor(scope: CategorizationControllerScope) {
         super(scope);
     }
-    public changeSelectedCategory(category) {
+    public changeSelectedCategory(category, clickScope) {
+      if (category.type === 'Category') {
         this.selectedCategory = category;
+      } else {
+        clickScope.expanded = !clickScope.expanded;
+        if (!clickScope.expanded) {
+          this.selectedCategory = null;
+        }
+      }
     }
 }
 
-const categorizationTemplate = `<jsonforms-layout>
+const categorizationTemplate = `
```

Comment:
Do we also need to update the material and bootstrap renderers? Running this with material looks a bit weird a `Sub` tab appear with not content.